components/Block.json

```diff
@@ -0,0 +1,193 @@
+{
+  "Number": {
```

Comment:
I don't think I understand the need for most of these parameters. You're aliasing basic integer types, the docs for `Block` should describe these elements, no?

Reply:
They do, yes, but the issue is how open RPC works and how that's then reused elsewhere. I think there are some things here that I made and then didn't use which I'll clean that up but in general this is a clean approach.